src/context/ChainApiContext.tsx

```diff
@@ -30,7 +29,7 @@ export function ChainApiContextProvider({children}: {children: React.ReactNode})
       return;
     }
 
-    queryClient.clear();
+    // queryClient.clear();
```

Comment:
We may not have to clear the cache on network switch now. Also it results to a better UX.
@franjs @ahoseinian please comment what you guys think.

Reply:
yes I guess it would be unnecessary.